airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/consts.py

```diff
@@ -64,4 +63,5 @@ class Formatter(Enum):
     ],
     Formatter.PYTHON: ["pyproject.toml", "poetry.lock"],
     Formatter.LICENSE: [LICENSE_FILE_NAME],
+    Formatter.JS: [".prettierrc", ".prettierignore"],
```

Comment:
If one of these files is missing, is that a no-op or an error? I'd assume (and hope) no-op, in which case we should probably list all documented filenames which are valid sources of configuration. For prettier (per [their docs](https://prettier.io/docs/en/configuration)), that list is:
> You can configure Prettier via (in order of precedence):
>
> A "prettier" key in your package.json file.
> A .prettierrc file written in JSON or YAML.
> A .prettierrc.json, .prettierrc.yml, .prettierrc.yaml, or .prettierrc.json5 file.
> A .prettierrc.js, or prettier.config.js file that exports an object using export default or module.exports (depends on the [type](https://nodejs.org/api/packages.html#type) value in your package.json).
> A .prettierrc.mjs, or prettier.config.mjs file that exports an object using export default.
> A .prettierrc.cjs, or prettier.config.cjs file that exports an object using module.exports.
> A .prettierrc.toml file.

The _full_ list does feel a little bit like overkill, but I also don't like the UX of our tool not picking up on valid configuration either, even in exotic cases. I do think we should, at minimum, also include `package.json`. I think it would be best to include the whole list, since it would preempt other `airbyte-ci`-only compatibility edge cases and since I don't expect that list to change--and so, by extension, the practical maintenance burden wouldn't be higher. I will grant that it is a little ugly, though:

```suggestion
    Formatter.JS: [
        "package.json",
        ".prettierrc",
        ".prettierignore"
        ".prettierrc",
        ".prettierrc.json",
        ".prettierrc.yml",
        ".prettierrc.yaml",
        ".prettierrc.json5",
        ".prettierrc.js",
        "prettier.config.js",
        ".prettierrc.mjs",
        "prettier.config.mjs",
        ".prettierrc.cjs",
        "prettier.config.cjs",
        ".prettierrc.toml",
    ],
```